src/store/pageview.ts

```diff
@@ -29,8 +47,9 @@ class ViewStore {
       ...metadata,
     };
     this.pageViews.push(pageView);
+    this._trim();
     this._save();
   }
 }
 
-export const viewStore = new ViewStore();
+export const viewStore = new ViewStore(300);
```

Comment:
We should always try and name our "magic" values.

`const DEFAULT_MAX_FEATURES_SIZE = 300`

Also I still am not sure why we need to pass this to the constructor? There is not reason why someone would make a new instance, that would actually break things - this has to be a singleton.

Reply:
I agree.